draft-ietf-rats-corim.md

```diff
@@ -996,6 +1000,36 @@ measurements for the Target Environment.
 {::include cddl/coswid-triple-record.cddl}
 ~~~
 
+#### Conditional Reference Series Triple {#sec-comid-triple-cond-ref}
+
+A Conditional Reference Series triple associates conditional Reference Value measurements,
+and optionally Endorsement measurements to a Target Environment, where the endorsed measurements
+are accepted if all of the reference measurements are accepted.
+
+The triple subject is a compound statement consisting of a Target Environment defined
+by `measurement-map`, and initial Reference Values, defined by `measurement-map`,
+that MUST be matched to Evidence before evaluating the triple object.
+If the initial Reference Values do not match Evidence, the triple is not matched
+and no Claims are accepted.
+
+The `authorized-by` value in `measurement-map`, if present, applies to all measurements
+in the triple including those in `conditional-series-record` records.
+
+The triple object is a series of additional measurements expressed as an array of
+`conditional-series-record` records that are evaluated in the order they appear in the array.
+The additional Reference Values, in each record, are evaluated against Evidence. If the
+Reference Values, in the first `measurement-values-map`, matches Evidence then the
+optional Endorsement values, in the second `measurement-values-map`, are also accepted.
+The first successfully matched record from the `conditional-series-record` array
+terminates evaluation retaining the accepted Claims.
+
+If none of the records in the series are matched then no Claims are accepted and
```

Comment:
I see that the term "claims" are used a few times. Wonder if it will be more clear to change all "endorsement measurements" to "endorsement claims"?

Reply:
A Claim is the combination of a named thing and its measurement(s). In this triple the triple subject is the named thing and the object is the set of measurements that are accepted. The measurement is more or less the triple object, but given a compound object like series records, the measurement could refer to either the reference or endorsed values. Given this understanding of Claim, does the proposed text still make sense?

Reply:
Yes, the existing proposed text still make sense.